### PR TITLE
Update password-rules.json

### PR DESCRIPTION
**Pull request by monishamandal02**
### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for websites-with-shared-credential-backends.json
- [x] There's evidence the domains are related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
- [x] The new group serves login pages on each of the included domains, and those login page accept accounts from the others. (For example, we don't associate `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for authentication.)

#### for change-password-URLs.json
- [x] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
- [x] The URL either makes the experience better or no worse than being directed to just the domain in a non-logged-in state

#### for password-rules.json
- [x] The given rule isn't particularly standard and obvious for password managers
- [x] Generated passwords have been tested from this rule using the [Password Rules Validation Tool](https://developer.apple.com/password-rules/)
- [x] Information has been included about the website's requirements (eg. screenshots, error messages, steps during experimentation, etc.)

### CODE REVIEW

**Comment by rmondello** [comment]
Can you provide some primary source information about the website and its rules? Did you experiment to come up with the rules, or were they stated clearly?

File: quirks/password-rules.json [hunk 1]
```diff
@@ -204,6 +204,9 @@
     "myaccess.dmdc.osd.mil": {
         "password-rules": "minlength: 9; maxlength: 20; required: lower; required: upper; required: digit; allowed: [@_#!&$`%*+()./,;~:{}|?>=<^'-[]];"
     },
+    "myntra.com": {
+        "password-rules": "minlength: 6; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@_#!&$`%*+()./,;~:{}|?>=<^'-[]];"
```
[the remark this answers is not in the document]
Comment (rmondello) [reply quirks/password-rules.json:208]: And even then, `-` and `]` have special placement treatment in character classes.

So this should be:

```
minlength: 6; allowed: lower, upper, digit, [-@_#!&$`%*+()./,;~:{}|?>=<^'[]];
```

File: quirks/password-rules.json [hunk 2]
```diff
@@ -204,12 +204,18 @@
     "myaccess.dmdc.osd.mil": {
         "password-rules": "minlength: 9; maxlength: 20; required: lower; required: upper; required: digit; allowed: [@_#!&$`%*+()./,;~:{}|?>=<^'-[]];"
     },
+    "myntra.com": {
+        "password-rules": "minlength: 6; allowed: required: lower; required: upper; required: digit; required: [!#$%&'()+,.:?@[_`~]];"
```
Comment (rmondello) [inline quirks/password-rules.json:208]: This isn’t a valid rule. Maybe you wanted?

```
minlength: 6; required: lower; required: upper; required: digit; required: [!#$%&'()+,.:?@[_`~]];
```

**Comment by rmondello** [comment]
I think we’re getting close with this one. I’d love to get this merged.

**Review by monishamandal02 — COMMENTED** [review]
Sorry for the late reply, I did the following changes.

**Comment by rmondello** [comment]
On July 15, I wrote:

> Can you provide some primary source information about the website and its rules? Did you experiment to come up with the rules, or were they stated clearly?

Can you do this? I’d love to get this merged.

**Pull request closed without merging**